event.py

```diff
@@ -0,0 +1,91 @@
+from typing import Iterable
+from copy import deepcopy
+from flask_restful import abort
+
+from timemodel import Time
+from eventdao import EventDAO
+
+
+DAYS = set(('Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'))
+
+class Event:
+    dao = EventDAO()
+
+    def __init__(self, name: str, days: Iterable[str], from_time: Time, to_time: Time, description=''):
+        assert set(days).issubset(DAYS)
+        assert from_time.in_seconds() <= to_time.in_seconds()
+
+        self.name = name
+        self.days = set(days)
+        self.from_time = from_time
+        self.to_time = to_time
+        self.description = description
+
+    @classmethod
+    def fromdict(cls, event_dict):
```

Comment:
No need for this method, as well, all the assignments will throw a KeyError exception if the key does not exist within the event_dict